src/Microsoft.IdentityModel.JsonWebTokens/LogMessages.cs

```diff
@@ -51,10 +51,11 @@ internal static class LogMessages
         internal const string IDX14107 = "IDX14107: Token string does not match the token formats: JWE (header.encryptedKey.iv.ciphertext.tag) or JWS (header.payload.signature)";
         internal const string IDX14111 = "IDX14111: JWT: '{0}' must have three segments (JWS) or five segments (JWE).";
         internal const string IDX14112 = "IDX14112: Only a single 'Actor' is supported. Found second claim of type: '{0}', value: '{1}'";
-
+        internal const string IDX14113 = "IDX14113: A duplicate value for 'SecurityTokenDescriptor.{0}' exists in 'SecurityTokenDescriptor.Payload'.";
```

Comment:
Can you add some text to say the value from the SecurityTokenDescriptor was used.

Reply:
Done. 👍